docs/clients/iquart_flight_controller_interface.rst

```diff
@@ -0,0 +1,132 @@
+IQUART Flight Controller Interface
+----------------------------------
+This client is used to simplify communication between the flight controller and multiple modules that are connected to an IQUART bus.
```

Comment:
used to simplify communication between flight controllers and multiple modules connected to the IQUART bus.